### ISSUE TITLE
Unable to create new variables 

### ISSUE THREAD

**Issue by nicholasareed** [envkey/envkey#29]
2nd time I've experienced this issue. Previously it went away when I restarted EnvKey, but now restarting won't fix the issue and I am unable to add new variables. 

I am trying to add my variable like normal, and when I click "Create Variable" nothing happens.

When I open the inspector, the console shows an error when I press Create Variable: 

```
Uncaught TypeError: Cannot read property 'canUpdate' of undefined
    at entry_form.tsx:89
    at Array.map (<anonymous>)
    at onClick (entry_form.tsx:83)
    at Object.Fe (react-dom.production.min.js:52)
    at Ve (react-dom.production.min.js:52)
    at react-dom.production.min.js:53
    at Mr (react-dom.production.min.js:100)
    at Er (react-dom.production.min.js:101)
    at react-dom.production.min.js:113
    at Pe (react-dom.production.min.js:292)
```

**Comment by danenania** [comment]
Hey @nicholasareed, a fix will be out shortly.

**Comment by danenania** [comment]
Should now be fixed as of 2.1.11
